app/assets/stylesheets/pages/nodes_list.scss

```diff
@@ -0,0 +1,28 @@
+.discovery-control {
```

Comment:
Maybe it is better if we use css file names that map to controllers. We can have a class on the body element of each page (generated by the current controller and current action) so inside the `app/assets/stylesheets/pages/nodes.scss` we will have the css for nodes#index nodes#show nodes#edit etc. I can send a commit that does that if you agree.